src/atom/projectClicked.ts

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+const projectClicked = atom({
+  key: 'projectClicked',
+  default: false,
+});
```

Comment:
보통 recoil사용할때, 한가지만 관리하는것이 아닐때가많아소,,!  저렇게 지정해주는게 좋고!!
recoil을 사용하는 이유가 전역 상태 관리이기때문에 페이지가 이동하면서 값을 유지시키는게 중요한데, 

그러려면 이 값을 만약에 유지시키고 싶다면 [Recoil-persist](https://velog.io/@timosean/Web-Recoil-persist-%EC%82%AC%EC%9A%A9%ED%95%B4%EB%B3%B4%EA%B8%B0)를 사용해주어야해!! 

```suggestion ts
const projectClicked = atom({
  key: 'projectClicked',
  default:    {
        isClicked: false,
  },
  effects_UNSTABLE: [persistAtom],
});
```

Reply:
므찌다 recoil....!! 나도 다음에 공부해서 사용해봐야지😂

Reply:
오와오아!! 그렇군!! 요거는 다시 써봐야겠다..!